lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -14,33 +11,18 @@
 from openedx.core.lib.grade_utils import is_score_higher
 from student.models import anonymous_id_for_user
 from submissions import api as submissions_api
-from traceback import format_exc
 from xmodule import block_metadata_utils, graders
 from xmodule.graders import AggregatedScore
 
 
 log = getLogger(__name__)
 
 
-@contextmanager
-def persistence_safe_fallback():
-    """
-    A context manager intended to be used to wrap robust grades database-specific code that can be ignored on failure.
-    """
-    try:
-        with transaction.atomic():
-            yield
-    except DatabaseError:
-        # Error deliberately logged, then swallowed. It's assumed that the wrapped code is not guaranteed to succeed.
-        # TODO: enqueue a celery task to finish the task asynchronously, see TNL-5471
-        log.warning("Persistent Grades: database_error.safe_fallback.\n{}".format(format_exc()))
-
-
 class SubsectionGrade(object):
     """
     Class for Subsection Grades.
     """
-    def __init__(self, subsection, course):
```

Comment:
This is the pylint error my editor uncovered - `course` is now an unused parameter and should be removed.